resources/assets/components/actions/SocialDriveAction/SocialDriveAction.js

```diff
@@ -10,6 +10,7 @@ import Card from '../../utilities/Card/Card';
 import Embed from '../../utilities/Embed/Embed';
 import { postRequest } from '../../../helpers/api';
 import { dynamicString, withoutTokens } from '../../../helpers';
+import { featureFlag } from '../../../helpers';
```

Comment:
The build is failing because we need to group this import in with the existing `/helpers` import above
```suggestion
import { dynamicString, featureFlag, withoutTokens } from '../../../helpers';
```